templates/index.html

```diff
@@ -477,13 +477,29 @@ <h4 class="panel-title">
                                 </h4>
                             </div>
 
-                            <div id="{{faqz.id}}" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
+                            {% comment %} <div class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
                                 <div class="panel-body">
                                     <p>{{faqz.answer}}</p>
                                 </div>
-                            </div>
+                            </div> {% endcomment %}
+
+                            {% for subqsn in subquestions%}
+                            {% if faqz.question == 'I have specific questions that are not addressed here. Who can help me ?' %}                           
+                            <div><b>{{subqsn.question}}</b></div>
+                            <div >{{subqsn.answer}}</div>
+                            {% endif %}
+
+                            {% comment %} # Note----> Elywiin use elseif here if u want to include a subquestion for another FAQ{% endcomment %}
```

Comment:
The idea is for anyone to use the system, so what we want is to be able to add a question and answer or answers through the admin dashboard, not hard coding coz some people don't know how to code and we want them to be able to use the system and populate it.